app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -13,15 +13,7 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
           N_('Show Capacity & Utilization data for this Server'),
           N_('Utilization'),
           :url       => "/show",
-          :url_parms => "?display=performance"),
-        button(
-          :middleware_server_timeline,
-          'product product-timeline fa-lg',
-          N_('Show Timelines for this Server'),
-          N_('Timelines'),
-          :enabled   => "false",
-          :url       => "/show",
-          :url_parms => "?display=timeline"),
+          :url_parms => "?display=performance")
```

Comment:
I'm a little confused...this seems to be removing the timeline...how is this related to CRUD operations?